src/terminal/parser/stateMachine.cpp

```diff
@@ -6,6 +6,7 @@
 #include "stateMachine.hpp"
 
 #include "ascii.hpp"
+#include "InputStateMachineEngine.hpp"

```

Comment:
You're going to hate this, but I think that there's an include order dependency... on account of the OneCoreSafe stuff?